torch_xla/csrc/runtime/sys_util.cpp

```diff
@@ -10,12 +10,12 @@ namespace torch_xla {
 namespace runtime {
 namespace sys_util {
 
-std::string GetEnvString(const char* name, const std::string& defval) {
-  const char* env = std::getenv(name);
+std::string GetEnvString(const char* const name, const std::string& defval) {
```

Comment:
`const std::string&` -> `std::string_view`?

Reply:
I'll save that for a future PR.